config/mock-api-data.js

```diff
@@ -31,6 +51,7 @@ module.exports = () =>
             // randomize prop is just a junk that enforces a different image per object
             img: `http://lorempixel.com/400/400/?randomize=${i}`,
             location: [Number(faker.address.latitude()), Number(faker.address.longitude())],
+            tag: setTagName(),
```

Comment:
IMO "fake" or "mock" should be used in function name to prevent any possibility of ambiguity.

Reply:
Fixed in afd0d43